var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -13,13 +13,58 @@ class Crtm(CMakePackage):
     scattering, and a solver for a radiative transfer."""
 
     homepage = "https://www.jcsda.org/jcsda-project-community-radiative-transfer-model"
-    url = "https://github.com/NOAA-EMC/EMC_crtm/archive/refs/tags/v2.3.0.tar.gz"
+    git = "https://github.com/JCSDA/crtm.git"
+    url = "https://github.com/JCSDA/crtm/archive/refs/tags/v2.3.0.tar.gz"
```

Comment:
To clarify, this was coming from a fork, and now it's not?

Reply:
Correct, now it's coming from the authoritative repo for CRTM, which is managed by @BenjaminTJohnson. Note that we've been using this code as-is for the last year in the noaa-emc spack fork. The authoritative repository never got updated and is seriously out of date.